app/views/application/_project_form_fields.html.haml

```diff
@@ -30,6 +30,13 @@
   .col-md-4
     = form.text_field :material_type, class: 'form-control', required: true
     .form-info Wool, Cotton, Linen, Synthetic (Nylon, Polyester, Acrylic, etc).
+- if current_user.can_manage?
+  .row.mb-4
```

Comment:
I think we may want to add a CSS class to any manager-only elements and visually differentiate them somehow. Just so managers know that the person they may be on the phone with cannot see the same thing.